### PR TITLE
Create cmssw.xml

### PR DESCRIPTION
**Pull request by AnxhelaDani**
CMSSW record for 2010

### CODE REVIEW

**Comment by pherterich** [comment]
@tiborsimko to add a record number, all the other fixmes will be taken care of in January

File: invenio_opendata/testsuite/data/cms/cmssw.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      <subfield code="u">http://opendatadev.cern.ch/getting-started/CMS</subfield>
+    </datafield>
+    <datafield tag="693" ind1=" " ind2=" ">
+      <subfield code="a">CERN-LHC</subfield>
+      <subfield code="e">CMS</subfield>
+    </datafield>
+    <datafield tag="581" ind1=" " ind2=" ">
+      <subfield code="a">See the run instructions in</subfield>
+      <subfield code="u">http://opendatadev.cern.ch/getting-started/CMS</subfield>
+      <subfield code="y">LINK NAMING</subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cmssw.xml]: Link naming?

File: invenio_opendata/testsuite/data/cms/cmssw.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+in the CMSSW WorkBook. Make sure, though, that you replace the release version (CMSSW_nnn) with the release that 
+you are using, i.e. one that is compatible with the CMS open data.  Be aware that the instructions in the WorkBook 
+are in use in CMS currently and have been updated for more recent CMSSW releases. With the 2010 data, you should 
+always use the releases in the series of CMSSW_4_2 and not higher. Also note that more recent code does not work with 
+older releases, so whenever you see git cms-addpkg… in the instruction, it is likely that the code package this command 
+adds does not work with the release you need.</subfield>
+      <subfield code="u">http://opendatadev.cern.ch/getting-started/CMS</subfield>
+    </datafield>
+    <datafield tag="583" ind1=" " ind2=" ">
+      <subfield code="a"> VALIDATION TEXT </subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cmssw.xml]: @AnxhelaDani The VALIDATION TEXT is still expanded.

**Comment by tiborsimko** [comment]
Thanks, squashed together, added record ID, renamed the file to stay close to other Tools files, and merged as:

- 7c51bbd861701b64d8eecc55069798a2d3a03cea CMS: CMS SW 2010 file name
- 870dbf6b2427bfd1ce89368a3d288b67040130f5 CMS: fix CDATA in CMS SW 2010 record
- 84effaa87717f89d4fdca1880072dc7906274fc4 CMS: addition of CMSSW record for 2010

**Pull request closed without merging**